README.md

```diff
@@ -1,178 +1,317 @@
-sass-spec
-=========
+# sass-spec
+
+### A cross-implementation [Sass][] test suite
+
+* [Running Specs](#running-specs)
+  * [Dart Sass](#dart-sass)
+  * [LibSass](#libsass)
+  * [Ruby Sass](#ruby-sass)
+* [Spec Structure](#spec-structure)
+  * [HRX](#hrx)
+  * [Specifying Warnings](#specifying-warnings)
+  * [Implementation-Specific Expectations](#implementation-specific-expectations)
+  * [Options](#options)
+    * [`:todo`](#todo)
+    * [`:warning_todo`](#warning_todo)
+    * [`:ignore_for`](#ignore_for)
+* [Organization](#organization)
+* [Interactive Mode](#interactive-mode)
+
+[Sass]: https://sass-lang.com
 
 [![Build Status](https://travis-ci.org/sass/sass-spec.svg)](https://travis-ci.org/sass/sass-spec)
 
-A test suite for Sass. The test cases are all in the `/spec` folder.
+`sass-spec` is the official Sass test suite. It's used by all major Sass
+implementations to ensure that they correctly implement the language.
 
-Sass spec is written in ruby, so you will need to have ruby and bundler
-installed in order to run it.
+## Running Specs
 
-## Running specs against Ruby Sass
+Before running specs, you'll need to install [Ruby][] and [Bundler][]. The spec
+runner is written in Ruby, so it's necessary no matter which implementation
+you're testing. Then, from the root of this repo, run `bundle install`.
 
-Run tests against Ruby Sass with the `sass-spec.rb` file in the root directory.
+[Ruby]: https://www.ruby-lang.org/en/
+[Bundler]: http://bundler.io/
 
-    $ git clone https://github.com/sass/sass-spec.git
-    $ cd sass-spec
-    $ bundle install
-    $ bundle exec sass-spec.rb
+From there, it depends which implementation you're testing:
 
-To run tests against Ruby Sass in development, edit the Gemfile and add:
+### Dart Sass
+
+To run specs against [Dart Sass][], the reference implementation of Sass that's
+used for [the `sass` package][] on npm, you'll first need to install [Dart][].
+Then run:
+
+[Dart Sass]: https://sass-lang.com/dart-sass
+[the `sass` package]: https://npmjs.com/package/sass
+[Dart]: https://www.dartlang.org/
+
+```sh
+# If you already have a clone of the Dart Sass repo, you can use that instead.
+git clone https://github.com/sass/dart-sass
+(cd dart-sass; pub get)
+export DART_SASS_PATH=`pwd`/dart-sass
```

Comment:
I think `$pwd` is more portable than backticks, but i'm not a bash expert

Reply:
Backquotes should work for any [`sh`-compliant shell](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03).

Reply:
Think I was coming more from https://github.com/koalaman/shellcheck#style but it looks like that s a style rather than bug rule https://github.com/koalaman/shellcheck/issues/364#issuecomment-97578330